app/notifications/validators.py

```diff
@@ -104,6 +104,11 @@ def warn_about_daily_message_limit(service, messages_sent):
                 },
                 include_user_fields=["name"],
             )
+            current_app.logger.info(
+                "service {} is approaching its daily limit, sent {} limit {}".format(
+                    service.id, int(messages_sent), service.message_limit
+                )
+            )
```

Comment:
indented here so we only get one log message per service per day (ie not for every notification)

Reply:
neat: can we move to f-strings a bit more in our code base? 🙏

Reply:
yep, good point. This was basically a copy-paste of the log for "over the daily limit" just below. I changed both logs to now be f-strings.